VirtoCommerce.CatalogModule.Web/Controllers/Api/CatalogModuleCatalogsController.cs

```diff
@@ -32,11 +32,12 @@ public CatalogModuleCatalogsController(ICatalogService catalogService, ICatalogS
 		[HttpGet]
         [Route("")]
         [ResponseType(typeof(webModel.Catalog[]))]
-        public IHttpActionResult GetCatalogs()
+        public IHttpActionResult GetCatalogs([FromUri]webModel.SearchOptions options)
```

Comment:
The formal parameter should be a type  SearchCriteria instead webModel.SearchOptions